app/experimenter/experiments/tests/test_forms.py

```diff
@@ -1317,6 +1336,86 @@ def test_optional_reviews_when_a_risk_option_is_true(self):
         self.assertNotIn(form["review_vp"], form.optional_reviews)
         self.assertIn(form["review_vp"], form.required_reviews)
 
+    def test_cannot_check_review_relman_without_permissions(self):
+        user_1 = UserFactory.create()
+        user_2 = UserFactory.create()
+
+        content_type = ContentType.objects.get_for_model(Experiment)
+        permission = Permission.objects.get(
+            content_type=content_type, codename="can_check_relman_signoff"
+        )
+        user_1.user_permissions.add(permission)
+        self.assertTrue(
+            user_1.has_perm("experiments.can_check_relman_signoff")
+        )
+        self.assertFalse(
+            user_2.has_perm("experiments.can_check_relman_signoff")
+        )
```

Comment:
I'm not sure if these assertions are really worth it.

Reply:
gotcha, I'll change.